app/views/layouts/shared/_user_menu.html.slim

```diff
@@ -1,23 +1,21 @@
-.collapse.navbar-collapse.float-right
-  ul.nav.navbar-nav.float-right
-    li.dropdown.dropdown-user.nav-item
-      a.dropdown-toggle.nav-link.dropdown-user-link[href="#" data-toggle="dropdown"]
-        span.avatar.avatar-online
-          = gravatar_image_tag(current_user.email)
-        span.user-name = current_user.full_name
-      .dropdown-menu.dropdown-menu-right
-        .dropdown-item[href="#"]
-          = link_to edit_user_registration_path, class: "btn btn-md" do
-            i.fa.fa-user
-            | Edit Profile
-        - if current_user != true_user
-          .dropdown-divider
-          .dropdown-item[href="#"]
-            = link_to stop_impersonating_account_users_path, method: :post, class: "btn btn-md" do
-              i.fa.fa-arrow-left
-              | Back to admin
+ul.nav.navbar-nav.float-right
+  li.dropdown.dropdown-user.nav-item
+    a.dropdown-toggle.nav-link.dropdown-user-link[href="#" data-toggle="dropdown"]
+      span.avatar.avatar-online
+        = gravatar_image_tag(current_user.email)
+        i
+      span.user-name
+        = current_user.full_name
+    .dropdown-menu.dropdown-menu-right
+      = link_to edit_user_registration_path, class: "dropdown-item" do
+        i.fa
+        | Edit Profile
+      - if current_user != true_user
```

Comment:
I guess it'll be better to put this condition into helper:  to avoid negative compare and maybe to reuse this someweher in future